types/slate/index.d.ts

```diff
@@ -1530,29 +1048,1079 @@ export interface PathUtils {
+     * Passing a string will set the nodes' type only.
+     */
+    setInlines(properties: InlineProperties | string): Editor;
+
+    /**
+     * Split the Block in the current selection by depth levels.
+     * If the selection is expanded, it will be deleted first.
+     */
+    splitBlock(depth: number): Editor;
+
+    /**
+     * Split the Inline node in the current selection by depth levels.
+     * If the selection is expanded, it will be deleted first
+     */
+    splitInline(depth: number): Editor;
+
+    /**
+     * Remove a mark from the characters in the current selection.
+     * Passing a string will implicitly create a Mark of that type for removal.
+     */
+    removeMark(mark: Mark | MarkProperties | string): Editor;
+
+    /**
+     * Remove a mark from the characters in the current selection.
+     * Passing a string will implicitly create a Mark of that type.
+     */
+    replaceMark(
+        mark: Mark | MarkProperties | string,
+        newMark: Mark | MarkProperties | string
+    ): Editor;
+
+    /**
+     * Add or remove a mark from the characters in the current selection, depending on it already exists on any or not.
+     * Passing a string will implicitly create a Mark of that type to toggle.
+     */
+    toggleMark(mark: Mark | MarkProperties | string): Editor;
+
+    /**
+     * Unwrap all Block nodes in the current selection that match a type and/or data
+     */
+    unwrapBlock(properties: BlockProperties | string): Editor;
+
+    /**
+     * Unwrap all Inline nodes in the current selection that match a type and/or data
+     */
+    unwrapInline(properties: InlineProperties | string): Editor;
+
+    /**
+     * Wrap the Block nodes in the current selection with a new Block
+     */
+    wrapBlock(properties: BlockProperties | string): Editor;
+
+    /**
+     *  Wrap the Block nodes in the current selection with a new Inline
+     */
+    wrapInline(properties: InlineProperties | string): Editor;
+
+    /**
+     * Surround the text in the current selection with prefix and suffix strings.
+     * If the suffix is ommitted, the prefix will be used instead.
+     */
+    wrapText(prefix: string, suffix?: string): Editor;
+
+    // Selection Commands //
+    /**
+     * Blur the current selection
+     */
+    blur(): Editor;
+
+    /**
+     * Unset the selection
+     */
+    deselect(): Editor;
+
+    /**
+     * Flip the selection
+     */
+    flip(): Editor;
+
+    /**
+     * Focus the current selection
+     */
+    focus(): Editor;
+
+    /**
+     * Move the anchor of the current selection backward n characters
+     */
+    moveAnchorBackward(n?: number): Editor;
+    /**
+     * Move the anchor of the current selection forward n characters
+     */
+    moveAnchorForward(n?: number): Editor;
+    /**
+     * Move the anchor of the current selection to a new path and offset
+     */
+    moveAnchorTo(path: Path, offset: number): Editor;
```

Comment:
A lot of these `move*To` methods actually also accept an `offset` as the first argument and the second argument is always optional.

Reply:
I was going based off of [this](https://github.com/ianstormtaylor/slate/blob/master/packages/slate/src/interfaces/range.js#L183) I see the offset is optional, so I will change that, however I don't see a way of provided an offset as the first argument, are you sure this is still the case?

Reply:
I guess that's fair based on what's represented there, but those are proxies to: https://github.com/ianstormtaylor/slate/blob/master/packages/slate/src/models/point.js#L299 (for instance)

Reply:
ah that's true, I'd personally prefer to leave it as it matches what is presented to the user in the docs